game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json

```diff
@@ -127,7 +137,8 @@
             "default_value": "CHANGEME",
```

Comment:
this should not have a default value

Reply:
thats default in the egg; 

This PR is only change the images; if you want further changes, then add it after merging